module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraCustomSecurityAttributeDefinitionAllowedValue.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Get-EntraCustomSecurityAttributeDefinitionAllowedValue.
+description: This article provides details on the Get-EntraCustomSecurityAttributeDefinitionAllowedValue command.
+
+ms.topic: reference
+ms.date: 07/23/2024
+ms.author: eunicewaweru
+ms.reviewer: stevemutungi
+manager: CelesteDG
+author: msewaweru
+external help file: Microsoft.Graph.Entra-Help.xml
+Module Name: Microsoft.Graph.Entra
+online version:
+schema: 2.0.0
+---
+
+# Get-EntraCustomSecurityAttributeDefinitionAllowedValue
+
+## SYNOPSIS
+
+Gets the predefined value for a custom security attribute definition.
+
+## SYNTAX
+
+### GetQuery (Default)
+
+```powershell
+Get-EntraCustomSecurityAttributeDefinitionAllowedValue 
+ -CustomSecurityAttributeDefinitionId <String>
+ [-Filter <String>] 
+ [<CommonParameters>]
+```
+
+### GetById
+
+```powershell
+Get-EntraCustomSecurityAttributeDefinitionAllowedValue 
+ -CustomSecurityAttributeDefinitionId <String>
+ -Id <String> 
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-EntraCustomSecurityAttributeDefinitionAllowedValue` cmdley gets the predefined value for a Microsoft Entra ID custom security attribute definition. Specify `CustomSecurityAttributeDefinitionId` parameter to get the predefined value custom security attribute definition.
+
+## EXAMPLES
+
+### Example 1: Get all predefined values
+
+```powershell
+Connect-Entra -Scopes 'CustomSecAttributeDefinition.Read.All, CustomSecAttributeDefinition.ReadWrite.All'
+Get-EntraCustomSecurityAttributeDefinitionAllowedValue -CustomSecurityAttributeDefinitionId 'Engineering_Project'
+```
+
+```Output
+Id    IsActive
+--    --------
+Apline True
+```
+
+This example Get all predefined values.
```

Comment:
Please add description of CustomSecurityAttributeDefinitionId in example section